IdentityCore/src/controllers/MSIDLocalInteractiveController.m

```diff
@@ -90,6 +90,30 @@ - (void)acquireToken:(MSIDRequestCompletionBlock)completionBlock
     }];
 }
 
+- (void)handleBrokerResponse:(MSIDWebMSAuthResponse *)response completion:(MSIDRequestCompletionBlock)completionBlock
+{
+    if (![NSString msidIsStringNilOrBlank:response.appInstallLink])
```

Comment:
Related to the above comment, handling broker response and prompting install is two different things, am i misunderstanding here?
Perhaps the method name is wrong then? I just don't see a broker response being "install broker"

Reply:
Changed naming, please take a look.